.orchestra/config/components/revng-vscode-web.yml

```diff
@@ -0,0 +1,70 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_vscode_web_args():
+binary_archives: private
+dependencies:
+- revng
+build_dependencies:
+- revng-ui
+- revng-theme
+- vscode-web
+configure: |
+  mkdir -p "$BUILD_DIR"
+install: |
+  PYTHON_PACKAGE_BASE=$("${ORCHESTRA_DOTDIR}/support/python-package-base")
+  COMMANDS_DIR="${PYTHON_PACKAGE_BASE}/revng/cli/_commands"
+  mkdir -p "$COMMANDS_DIR"
+  cp -a "${ORCHESTRA_DOTDIR}/support/vscode-web/launch-vscode-web.py" \
+        "$COMMANDS_DIR"
+
+  INSTALL_DIR="${DESTDIR}${ORCHESTRA_ROOT}/share/vscode-web"
+  mkdir -p "$INSTALL_DIR/static"
+  tar -xf "${ORCHESTRA_ROOT}/share/vscode-web.tar.xz" -C"$INSTALL_DIR"
+
+  # Repackage vsixs inside the destionation directory
+  # If the EXTENSIONS_DIR has been specified and exists we'll take each vsix file and unzip it,
+  # as this is the format needed by vscode-web to load extensions properly
+  for extension in "$ORCHESTRA_ROOT"/share/vscode-extensions/*.vsix; do
+    # We need a temporary directory since:
+    # * unzip does not have the equivalent of --strip-components
+    # * we need to read the extension's name from the package.json file
+    TEMP=$(mktemp -d --tmpdir tmp.revng.vscode-web-ext-unpack.XXXXXXXXXX)
```

Comment:
```suggestion
    TEMP=$(mktemp -d tmp.revng.vscode-web-ext-unpack.XXXXXXXXXX)
```

Reply:
without `--tmpdir` the cwd is used to create the temporary directory

Reply:
Uhm, but you no longer have a template then.
Maybe you meant `mktemp --tmpdir ${TMPDIR:-/tmp} tmp.revng.vscode-web-ext-unpack.XXXXXXXXXX`?

Reply:
```
-p DIR, --tmpdir[=DIR]
              interpret TEMPLATE relative to DIR; if DIR is not specified, use $TMPDIR if set, else /tmp.  With this option, TEMPLATE must not be an  ab‐
              solute name; unlike with -t, TEMPLATE may contain slashes, but mktemp creates only the final component
              ```
              
If you omit the `DIR` parameter it will do the right thing™

Reply:
Ah-ah gotcha.
We'd need to do this everywhere.